admin/packaging.py

```diff
@@ -910,6 +910,8 @@ def omnibus_package_builder(
                      flocker_node_path),
                     (FilePath('/opt/flocker/bin/flocker-dataset-agent'),
                      flocker_node_path),
+                    (FilePath('/opt/flocker/bin/flocker-diagnostics'),
+                     flocker_node_path),
```

Comment:
@jml These are the stupid tests we were discussing yesterday....I could remove them rather than making them uglier. What do you think?